### DISCUSSION TITLE
发送消息的API问题（已经解决）

### DISCUSSION THREAD

**Discussion by 2926295173** [Mrs4s/go-cqhttp#1123] [Q&A]
我开启了反向websockt.但是我用ws协议向

/send_private_msg接口
发送私聊的数据，显示api不存在，
请问，像以上那种api只能通过开启http后用request库来post嘛？

**Comment by 2926295173** [comment]


**Reply by huamushen1624** [reply]
大佬解决了吗,这里也碰到一样的问题,属实不知道反向ws该怎么调api了
